flux-config/src/lib.rs

```diff
@@ -78,6 +82,7 @@ struct Config {
     check_def: String,
     cache: bool,
     cache_file: String,
+    check_overflow: bool,
```

Comment:
The option in `rustc` to enable disable overflow checks is `-Coverflow-checks`. Not sure if we should be consistent with that and call the flag `FLUX_OVERFLOW_CHECKS`. I think I generally prefer `check overflows` as if you are telling the tool `do (not) check overflows`, but maybe being consistent is better. We can always change it later though, so no biggie.

Reply:
We're already diverging with Rust with the way we handle overflow checking, right? I agree that our name is more consistent (both with intuition and also the naming convention of our flags).